lib/ret/application.ex

```diff
@@ -14,9 +14,29 @@ defmodule Ret.Application do
     repos_pids = EctoBootMigration.start_repos([Ret.Repo])
 
     Ret.Locking.exec_if_session_lockable("ret_migration", fn ->
+      db_name = Application.get_env(:ret, Ret.Repo)[:database]
+
       # Can't check mix_env here, so check db name
-      if Application.get_env(:ret, Ret.Repo)[:database] !== "ret_test" do
+      if db_name !== "ret_test" do
+        coturn_enabled = Ret.Coturn.enabled?()
+
         Ecto.Adapters.SQL.query!(Ret.Repo, "CREATE SCHEMA IF NOT EXISTS ret0")
+
+        if coturn_enabled do
+          Ecto.Adapters.SQL.query!(Ret.Repo, "CREATE SCHEMA IF NOT EXISTS coturn")
+        end
+
+        Ecto.Adapters.SQL.query!(
+          Ret.Repo,
+          "ALTER DATABASE #{db_name} SET search_path TO ret0#{
+            if coturn_enabled do
+              ", coturn"
+            else
+              ""
+            end
+          }"
```

Comment:
nit: this would be a bit easier to read if you had a list of schemas and joined them with a comma.